.github/workflows/ui-tests.yml

```diff
@@ -63,22 +66,25 @@ jobs:
 
   # SwiftUI only supports iOS 14+ so we run it in a separate matrix here
   ui-tests-swift-ui:
-    name: UI Tests for SwiftUI on ${{matrix.device}} Simulator
```

Comment:
`m`: Either leave any value of the matrix here, so they are not named identically when adding new matrix values, or remove the matrix entirely

Reply:
Good point. I removed the matrix.